src/nxdoc/nuxeo-add-ons/nuxeo-for-adobe-cc.md

```diff
@@ -347,7 +364,7 @@ If you want to edit an existing file from your Nuxeo account:
     ![](nx_asset://d435ccfb-4624-4e6f-8ca9-91c440b856b9 ?w=350,border=true)
 
 
-When edits are made to repository content using the Connector, users in InDesign who have placed the content will be notified that updates are available.
+When edits are made to repository content using the connector, users in InDesign who have placed the content will be notified that updates are available.
```

Comment:
The references to "the connector" in the previous sections have this word capitalised.